README.md

```diff
@@ -22,28 +87,41 @@ Contents
 npm install --save-dev @manta-style/cli
 ```
 
-You could also install it globally, which adds a command line tool `ms` to your system.
+To install globally
```

Comment:
according to the discussion this afternoon, we might not support global install at the moment.